lib/cli.coffee

```diff
@@ -25,12 +26,20 @@ installMycha = (testHelperPath, done) ->
   installer.install done
 
 
+loadConfig = (done) ->
```

Comment:
ok. What do you think about the other idea in my code snippet, exporting only the `load` method directly, instead of a hash {load: loadmethod}?